### PR TITLE
fancy nickname generator # 2

### PR DESCRIPTION
**Pull request by Chemoday**
forgot to add file in previous one

### CODE REVIEW

File: examples/simpleMVC/src/com/domwires/example/simplemvc/commands/GenerateNickNameCommand.as [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ */
+package com.domwires.example.simplemvc.commands
+{
+	import com.domwires.core.mvc.command.AbstractCommand;
+	import com.domwires.example.simplemvc.model.IAppModel;
+	import com.domwires.example.simplemvc.utils.RandomStringUtil;
+
+	public class GenerateNickNameCommand extends AbstractCommand
+	{
+		[Autowired]
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/commands/GenerateNickNameCommand.as:12]: improve code format

File: examples/simpleMVC/src/com/domwires/example/simplemvc/commands/GenerateNickNameCommand.as [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+		override public function execute():void
+		{
+			super.execute();
+
+            var nickname:String;
+            var random_name:Number;
+            var random_surname:Number;
+            var name:Array = new Array("Hayman ","Junious ","Nathaniel ","Alvin ","Lashley ","Rolandus ", "Rolandus", "Albany");
+            var surname:Array = new Array("Beemer","Ingles","Geffer","Pariet","Knowlton","Riddel", "Todd");
+            random_name = Math.floor(Math.random() * name.length);
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/commands/GenerateNickNameCommand.as]: do not use "_" in local variables. Example according language conventions: "randomName"

File: examples/simpleMVC/src/com/domwires/example/simplemvc/model/AppModel.as [hunk 3]
```diff
@@ -9,6 +9,7 @@ package com.domwires.example.simplemvc.model
 	{
 		private var _firstName:String;
 		private var _lastName:String;
+        private var _nickName:String;
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/model/AppModel.as]: improve code format

File: examples/simpleMVC/src/com/domwires/example/simplemvc/model/AppModel.as [hunk 4]
```diff
@@ -38,6 +39,15 @@ package com.domwires.example.simplemvc.model
 			return this;
 		}
 
+        public function setNickName(value:String):IAppModel
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/model/AppModel.as]: improve code format

File: examples/simpleMVC/src/com/domwires/example/simplemvc/model/AppModel.as [hunk 5]
```diff
@@ -48,6 +58,11 @@ package com.domwires.example.simplemvc.model
 			return _lastName;
 		}
 
+        public function get nickName():String
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/model/AppModel.as]: improve code format

File: examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as [hunk 6]
```diff
@@ -42,12 +43,14 @@ package com.domwires.example.simplemvc.view
 			firstNameLabel = createForm("First name:", layoutGroup, onFirstNameClicked);
 			lastNameLabel = createForm("Last name:", layoutGroup, onLastNameClicked);
 			ageLabel = createForm("Age:", layoutGroup, onAgeClicked);
+            nickNameLabel = createForm("Nickname:", layoutGroup, onNickNameClicked);
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as]: improve code format

File: examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as [hunk 7]
```diff
@@ -65,6 +68,11 @@ package com.domwires.example.simplemvc.view
 			firstNameLabel.text = model.firstName;
 		}
 
+		private function nickNameChanged(m:IMessage):void
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as:71]: improve code format

File: examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as [hunk 8]
```diff
@@ -106,5 +114,12 @@ package com.domwires.example.simplemvc.view
 		{
 			dispatchMessage(AppViewMessage.AGE_CLICKED, null, true);
 		}
+
+		private function onNickNameClicked():void
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as:118]: improve code format

**Review by CrazyFlasher — CHANGES_REQUESTED** [review]
Attach screenshot of final build

**Comment by Chemoday** [comment]
project structure : http://joxi.ru/L210ovi65xJBmX
program : http://joxi.ru/ZrJXMNT1qlga2j

**Comment by CrazyFlasher** [comment]
Please, resolve conflicts

**Pull request closed without merging**